cmd/cobra_types.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+)
+
+type PoolIds struct {
+	value *[]uint64
```

Comment:
Is there a particular reason `value` is a pointer to a slice instead of simply a slice?

Reply:
It's something I've also learnt myself, yesterday. 
My idea was to integrate a custom flag to be used like the others and to keep the flow of the code in line with your way of coding. So I checked how was the cobra implementation under the hoods of the other flags (slice) and I noticed that it was always used a pointer to a slice. 
It seems that by not using a pointer, if the underlying array gets reallocated (ex. using append), its value is reflected BUT the metadata len, cap might reflect the previous allocation. At least this is what I got from this also: https://medium.com/swlh/golang-tips-why-pointers-to-slices-are-useful-and-how-ignoring-them-can-lead-to-tricky-bugs-cac90f72e77b